rotkehlchen/tests/api/test_accounting_rules.py

```diff
@@ -0,0 +1,132 @@
+import requests
+from rotkehlchen.accounting.structures.types import HistoryEventSubType, HistoryEventType
+
+from rotkehlchen.tests.utils.api import (
+    api_url_for,
+    assert_proper_response_with_result,
+    assert_simple_ok_response,
+)
+
+
+def test_manage_rules(rotkehlchen_api_server):
+    """"
+    Test basic operations in the endpoint for managing accounting rules
+    """
```

Comment:
If it fits keep it in one line

```suggestion
    """"Test basic operations in the endpoint for managing accounting rules"""
```